lib/daru/dataframe.rb

```diff
@@ -244,6 +244,17 @@ def from_vector_rows source, opts
     #   #  b          7          2
     #   #  c          8          3
     #   #  d          9          4
+    #
+    #   df = Daru::DataFrame.new([[1,2,3,4],[6,7,8,9]], name: :bat_man)
+    #
+    #   # =>
+    #   # #<Daru::DataFrame: bat_man (4x2)>
+    #   #             0          1
```

Comment:
I hadn't used the actual executed code in the initial commit (I had copy-pasted the previous `rdoc` example - my bad). I've added the output I get in the `irb` console while initializing the DataFrame given in the `rdoc` example, in the subsequent commit.